test/rekt/features/apiserversource/data_plane.go

```diff
@@ -351,6 +352,84 @@ func SendsEventsForAllResources() *feature.Feature {
 	return f
 }
 
+func SendsEventsForAllResourcesWithNamespaceSelector() *feature.Feature {
```

Comment:
I think we should also test the "all namespaces" use case (maybe once we have the final API shape of it)

Reply:
I added another test for all namespaces. Since we assert with the pod name, it shouldn't matter if we're watching the system namespaces.